python/pyarrow/_dataset_parquet.pyx

```diff
@@ -67,11 +67,11 @@ cdef class ParquetFileFormat(FileFormat):
     Parameters
     ----------
     read_options : ParquetReadOptions
-        Read options for the file.
+        Read options for the file
```

Comment:
This one is puzzling though. Why remove this period?

Reply:
Not sure what I was doing there. I'll add the period back in.